openmetadata-ui/src/main/resources/ui/src/pages/teams/TeamsPage.tsx

```diff
@@ -474,6 +482,58 @@ const TeamsPage = () => {
     setShowDeletedTeam(checked);
   };
 
+  const fetchAssets = () => {
+    searchData(
+      `owner.id:${selectedTeam.id}`,
+      assets.currPage,
+      LIST_SIZE,
+      ``,
+      '',
+      '',
+      myDataSearchIndex
+    )
+      .then((res: SearchResponse) => {
+        const hits = res?.data?.hits?.hits;
+        if (hits?.length > 0) {
+          const data = formatDataResponse(hits);
+          const total = res.data.hits.total.value;
+          setSssets({
+            data,
+            total,
+            currPage: assets.currPage,
+          });
+        } else {
+          const data = [] as FormattedTableData[];
+          const total = 0;
+          setSssets({
+            data,
+            total,
+            currPage: assets.currPage,
+          });
+        }
+      })
+      .catch((err: AxiosError) => {
+        showErrorToast(
+          err,
+          jsonData['api-error-messages']['elastic-search-error']
```

Comment:
Let's use localisation instead of jsonData now

Reply:
Done